### ISSUE TITLE
Problems with the installation

### ISSUE THREAD

**Issue by NiteElSamu** [AkshitIreddy/Interactive-LLM-Powered-NPCs#13]
I need help, I'm trying to install both by downloading the directory and by following the commands step by step but when using git bash I get this error, I need help
![imagen_2024-12-04_213805962](https://github.com/user-attachments/assets/dacc6774-c335-483c-8096-0268d56f1724)

**Comment by sinaflair** [comment]
did you install wget?

**Comment by AkshitIreddy** [comment]
did you install wget?

**Comment by NiteElSamu** [comment]
No, I was following the tutorial and nowhere does it mention that you have to download wget.

**Comment by AkshitIreddy** [comment]
https://github.com/AkshitIreddy/Interactive-LLM-Powered-NPCs?tab=readme-ov-file#-wget

**Comment by DuckersMcQuack** [comment]
https://gnuwin32.sourceforge.net/packages/wget.htm

Install this

Then add `C:\Program Files (x86)\GnuWin32\bin` to path
Re-launch anaconda and try again.
